images/10-waagent/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM debian:stable-slim
+
+RUN apt-get update && apt-get install -yq --no-install-recommends \
+        ifupdown2 \
+        ca-certificates \
+        openssl \
+        sudo \
+        net-tools \
+        parted \
+        iptables \
+        eject \
+        host \
+        python-setuptools \
+        openssh-server && \
+        apt-get clean && rm -rf /var/lib/apt/lists/*
+
+COPY v2.2.34.tar.gz .
+
+RUN tar xvzf v2.2.34.tar.gz && \
+    cd WALinuxAgent-2.2.34 && \
+    sudo python setup.py install && \
+    cd .. && \
+    rm -rf WALinuxAgent-2.2.34 && \
+    rm -rf v2.2.34.tar.gz
+
+COPY waagent.conf /etc/waagent.conf
```

Comment:
I can see this `waagent.conf` in the tar file.
Do we need to copy from the outside?

Reply:
Already modified